### PR TITLE
creates operational process for force syncing for crm

### PR DESCRIPTION
**Pull request by saikumar9**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [x] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [x] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: [Pod of War - 187604366](https://www.pivotaltracker.com/story/show/187604366)

# A brief description of the changes

Current behavior: N/A

New behavior: Adds a script and a couple of operations to trigger CRM force sync for a given primary person hbx_ids.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [x] ME

# Additional Context

```ruby
# Command to trigger the script:
#   bundle exec rails runner script/crm/trigger_force_sync.rb 'hbx_id1, hbx_id2, hbx_id3'
```

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/domain/operations/crm/family/publish.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+        # Creates an event source event for the family entity.
+        #
+        # @param family_entity [FamilyEntity] The family entity object.
+        # @param headers [Hash] The headers for the event.
+        # @return [Dry::Monads::Result] The result containing the event.
+        def create_es_event(family_entity, headers)
+          event(
+            'events.families.created_or_updated',
+            attributes: { before_save_cv_family: {}, after_save_cv_family: family_entity.to_h },
```
Comment (jacobkagon) [inline app/domain/operations/crm/family/publish.rb:95]: @saikumar9 we're passing in cv family payload, not family entity. Is it ok to pass in family entity here?

Comment (saikumar9) [reply app/domain/operations/crm/family/publish.rb:95]: > @saikumar9 we're passing in cv family payload, not family entity. Is it ok to pass in family entity here?

I am converting the family_entity to a hash. So, the family payload is sent to the event source event, not the entity.

**Review by jacobkagon — APPROVED** [review]
Looks good. Creates new operation to manually trigger crm sync

**Review by jayreddy519 — APPROVED** [review]
lgtm.

**Pull request merged**